Backend/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "chase-car-dashboard-backend",
-  "version": "1.1.2",
```

Comment:
Do we have versioning guidelines? Like what defines a major or minor or minor minor change?